Documentation/network/concepts/ipam/cluster-pool.rst

```diff
@@ -45,6 +45,13 @@ Configuration
 For a practical tutorial on how to enable this mode in Cilium, see
 :ref:`gsg_ipam_crd_cluster_pool`.
 
+.. note::
```

Comment:
Does this need to use a `.. note::` or can we just create a dedicated section title or just put the text directly in without a callout box?

It is very common for contributors who are new to the docs to think that they're just adding one more note to the docs so it should be separate like this, but if we don't guard against this then eventually every single page is just filled with `..note` boxes that draw user attention to them.

This specific change seems to be related to "Expanding the cluster pool", so maybe this text should just be under a new title section with that title?
```suggestion
Expanding the cluster pool
==========================
```

Reply:
I thought about that, I even started with putting a warning before. I'm with you that a new section is a better idea.